pkgs/top-level/all-packages.nix

```diff
@@ -31806,4 +31806,9 @@ in
   zthrottle = callPackage ../tools/misc/zthrottle { };
 
   zktree = callPackage ../applications/misc/zktree {};
+
+  veikk-linux-driver-gui = callPackage ../os-specific/linux/veikk/gui {
```

Comment:
Please sort this into a fitting category and sort there alphabetically.

Reply:
Moved into pkgs/tools/misc/